src/Framework/TestCase.php

```diff
@@ -1285,16 +1297,23 @@ private function runTest(): mixed
         } finally {
             if ($capture !== false) {
                 fclose($capture);
-            }
 
-            ini_set('error_log', $errorLogPrevious);
+                /** @phpstan-ignore variable.undefined (https://github.com/phpstan/phpstan/issues/12992) */
+                ini_set('error_log', $errorLogPrevious);
```

Comment:
instead of ignoring a PHPStan error here, I would initialize the variable with null before the try/catch

Reply:
I prefer it this way as LoC is the same and once the PHPStan issue is implemented, PHPStan would fail and the ignore can be easily removed.